packages/frontend/src/lib/Badge.svelte

```diff
@@ -2,9 +2,13 @@
 import type { IconDefinition } from '@fortawesome/free-regular-svg-icons';
 import Fa from 'svelte-fa';
 
-export let icon: IconDefinition;
-export let content: string;
-export let background: string = 'bg-[var(--pd-label-bg)]';
+interface Props {
+  icon?: IconDefinition;
```

Comment:
icon does not have a default value, shouldn't it be required?

Reply:
Same concern

Reply:
On the non-svelte 5, it is mandatory, and should stay like that :+1: